pkg/metrics/collector.go

```diff
@@ -47,6 +47,31 @@ var (
 			return vs.UnknownCount
 		},
 	}
+	configAuditLabels = []string{
+		"namespace",
+		"name",
+		"severity",
+	}
+	configAuditDesc = prometheus.NewDesc(
+		prometheus.BuildFQName("trivy", "resource", "configaudit"),
```

Comment:
```suggestion
		prometheus.BuildFQName("trivy", "resource", "configaudits"),
```
? To make it match`_vulnerabilities` from the other metrics?

Reply:
Yes lets make it plural

Reply:
Updated